### PR TITLE
USSD Public functionality - Change number

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

**Comment by gsvr** [comment]
@NicolaasDieBaas Please review.

File: src/ussd_public.js [hunk 1]
```diff
@@ -400,17 +414,69 @@ go.app = function() {
...
+                }
+            });
+        });
+
+        // Interstitial
+        self.add('state_update_number', function(name, creator_opts) {
+            return go.utils
+                .get_identity(self.im.user.answers.contact_id, self.im)
+                .then(function(contact) {
+                    // TODO: Handle multiple addresses, currently overwrites existing
```
Comment (imsickofmaps) [inline src/ussd_public.js]: Can we capture these outside the code as issues please

**Comment by imsickofmaps** [comment]
Capture the todo's then :+1:

**Pull request merged**